### PR TITLE
Support sending auditing & events logs to loki

### PR DESCRIPTION
**Pull request by benjaminhuo**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:
Fixes #<issue number>

**Special notes for your reviewer**:

**Checklist**
- [ ] Reviewed the [`CONTRIBUTING.md`](https://github.com/grafana/loki/blob/main/CONTRIBUTING.md) guide (**required**)
- [ ] Documentation added
- [ ] Tests updated
- [ ] Title matches the required conventional commits format, see [here](https://www.conventionalcommits.org/en/v1.0.0/)
  - **Note** that Promtail is considered to be feature complete, and future development for logs collection will be in [Grafana Alloy](https://github.com/grafana/alloy). As such, `feat` PRs are unlikely to be accepted unless a case can be made for the feature actually being a bug fix to existing behavior.
- [ ] Changes that require user attention or interaction to upgrade are documented in `docs/sources/setup/upgrade/_index.md`
- [ ] For Helm chart changes bump the Helm chart version in `production/helm/loki/Chart.yaml` and update `production/helm/loki/CHANGELOG.md` and `production/helm/loki/README.md`. [Example PR](https://github.com/grafana/loki/commit/d10549e3ece02120974929894ee333d07755d213)
- [ ] If the change is deprecating or removing a configuration option, update the `deprecated-config.yaml` and `deleted-config.yaml` files respectively in the `tools/deprecated-config-checker` directory. [Example PR](https://github.com/grafana/loki/pull/10840/commits/0d4416a4b03739583349934b96f272fb4f685d15)

### CODE REVIEW

**Comment by benjaminhuo** [comment]
cc @wanjunlei

File: production/helm/whizard-loki/vector-agent-auditing-sink-loki.yaml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+        endpoint: http://loki-gateway.loki.svc
+        path: /loki/api/v1/push
+        tenant_id: "{{ .cluster }}"
+        out_of_order_action: accept
+        encoding:
+          codec: json
+        request:
+          retry_attempts: 5
+        labels: 
+          type: auditing
```
Comment (wanjunlei) [inline production/helm/whizard-loki/vector-agent-auditing-sink-loki.yaml:26]: 这些label会把stream分的特别小，loki中会存在大量的stream，每个stream里面只有很少的数据，对象存储中会存在大量的小文件，在查询的时候会对性能造成影响

**Comment by wanjunlei** [comment]
对接loki以文档形式提供吗？不需要集成到插件里？

**Comment by benjaminhuo** [comment]
> 对接loki以文档形式提供吗？不需要集成到插件里？

需要做成扩展，这里只是把如何配置 loki 写到这，到时候做插件参考

File: production/helm/whizard-loki/vector-agent-auditing-sink-loki.yaml (hunk 1 above)
Comment (benjaminhuo) [reply production/helm/whizard-loki/vector-agent-auditing-sink-loki.yaml:26]: 去掉了 resource name，其他的还是有必要的
image

File: production/helm/whizard-loki/vector-agent-auditing-sink-loki.yaml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+stringData:
+  sink.yaml: |
+    sinks:
+      loki-auditing:
+        type: loki
+        inputs:
+        - auditing_remapped
+        endpoint: http://loki-gateway.loki.svc
+        path: /loki/api/v1/push
+        tenant_id: whizard-auditing-ks
```
Comment (benjaminhuo) [inline production/helm/whizard-loki/vector-agent-auditing-sink-loki.yaml:19]: auditing 和 events 放到了单独的 tenant，便于单独设置 retention
image

**Pull request merged**